timemachine/fe/single_topology.py

```diff
@@ -681,7 +681,9 @@ def interpolate_harmonic_angle_params(src_params, dst_params, lamb, k_min, lambd
         lamb,
     )
 
-    return jnp.array([k, phase])
+    eps = jnp.where((lamb == 0.0) | (lamb == 1.0), 0.0, 1e-3)
```

Comment:
Should we have a constant for this epsilon value? Not sure how much giving it a different value matters or not.

Reply:
This was chosen somewhat arbitrarily to be large enough that no numerical instabilities were observed, and small enough to minimize the impact on end-state overlaps. Maybe we could add a comment to that effect? (Seemed premature to declare a constant, since I think this part of the code is relatively likely to change and it seemed useful to keep the definition local)

Reply:
A comment would work for me

Reply:
Added a comment in https://github.com/proteneer/timemachine/pull/938/commits/e52aa61342ef43b1d913fdfc3941e25713ee08ed